shared/src/store/action-reducers/hospital.ts

```diff
@@ -137,7 +137,7 @@ export namespace HospitalActionReducers {
                 logVehicle(
                     draftState,
                     [createHospitalTag(draftState, hospitalId)],
-                    'Ein Fahrzeug hat ein Krankenhaus angefahren',
+                    `${vehicle.name} hat ein Krankenhaus angefahren`,
```

Comment:
1. Why did you change it only here and not for all the other log messages?
2. Why do you query for the vehicle name instead of letting the helper do this? (The way you're doing it right now requires having an instance of the vehicle while for doing it in the helper, the vehicle id would suffice.)